cmd/spire-agent/cli/run/run.go

```diff
@@ -426,6 +426,10 @@ func NewAgentConfig(c *Config, logOptions []log.Option, allowUnknownConfig bool)
 		ac.LogReopener = log.ReopenOnSignal(logger, reopenableFile)
 	}
 
+	if c.Agent.RetryBootstrap {
+		ac.Log.Warn("The 'retry_bootstrap' configuration is deprecated. It will be enabled by default in SPIRE 1.13 and the option removed in SPIRE 1.14.")
+	}
+
```

Comment:
We are unable to change default behaviors without a complete minor rotation, so we need to move this to 1.14, before removing, 
And need to display this to users that are not using it.
Maybe something like this?

```suggestion
	if !c.Agent.RetryBootstrap {
		ac.Log.Warn("The 'retry_bootstrap' configuration is going to be default in SPIRE 1.14 and the option removed in SPIRE 1.15.")
	}

```